app/serializers/transcription_serializer.rb

```diff
@@ -1,6 +1,9 @@
 class TranscriptionSerializer
   include FastJsonapi::ObjectSerializer
 
-  attributes :workflow_id, :group_id, :text, :status, :flagged, :updated_at, :updated_by
+  attributes :workflow_id, :group_id, :status, :flagged, :updated_at, :updated_by
+  attribute :text, if: proc { |_record, params|
+    params && params[:is_collection] == false
```

Comment:
Is this always true or does it depend on the route? Is it safe to say that we never want to include the text on collection routes? Or should it be an option? A question for WIll or Sam that may have already been answered, just seems a little cut n' dry here.

Reply:
For instance, I just saw some code for filtering transcriptions by internal ID. You'd only get one back, but you'd get it as a collection, without text.

Reply:
Ahh, you are right, I will confirm 👍